pkg/sif/descriptor.go

```diff
@@ -295,6 +300,24 @@ func (d Descriptor) SBOMMetadata() (SBOMFormat, error) {
 	return s.Format, nil
 }
 
+// OCIBlobMediaType returns the media for a OCI blob object.
+func (d Descriptor) OCIBlobMediaType() (string, error) {
+	if got, want := d.raw.DataType, DataOCIBlob; got != want {
+		return "unknown", &unexpectedDataTypeError{got, []DataType{want}}
+	}
+
+	var o ociBlob
+	if err := d.raw.getExtra(binaryUnmarshaler{&o}); err != nil {
+		return "unknown", fmt.Errorf("%w", err)
+	}
+
+	if str := string(bytes.TrimRight(o.MediaType[:], "\x00")); str != "" {
+		return str, nil
+	}
+
+	return "unknown", nil
```

Comment:
Let's return `""` if we don't have a media type. Who knows, maybe someone will want to literally put `"unknown"` in the field at some point...

```suggestion
	if got, want := d.raw.DataType, DataOCIBlob; got != want {
		return "", &unexpectedDataTypeError{got, []DataType{want}}
	}

	var o ociBlob
	if err := d.raw.getExtra(binaryUnmarshaler{&o}); err != nil {
		return "", fmt.Errorf("%w", err)
	}

	if str := string(bytes.TrimRight(o.MediaType[:], "\x00")); str != "" {
		return str, nil
	}

	return "", nil
```